apps/docs/content/docs/sync.mdx

```diff
@@ -313,3 +206,22 @@ async function loadOrMakeRoom(roomId: string) {
 	return new TLSocketRoom()
 }
 ```
+
+## WebSocket server
```

Comment:
it's weird to me that this is way at the bottom here. it's like the core part of our multiplayer server?